domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaim.java

```diff
@@ -88,6 +90,17 @@
     private ClaimDocumentCollection claimDocumentCollection = new ClaimDocumentCollection();
     private LocalDate claimantResponseDeadline;
     private ClaimState state = null;
+    public static Supplier<ClaimSubmissionOperationIndicators> getDefaultClaimSubmissionOperationIndicators =
```

Comment:
I don't see any usage of this outside of this class, `private` should be enough. Do we need to make this static at all? Either choose `private` field or `public static final`.

Reply:
Made private. Brought it from external file, left the access specifier as private.